src/components/v5/shared/CustomScrollbar/CustomScrollbar.module.css

```diff
@@ -0,0 +1,11 @@
+.customScrollbar::-webkit-scrollbar {
```

Comment:
we discussed to not create separate files with tailwind classes, please add it global styles

Reply:
@CzarekDryl Maybe we could use this https://github.com/adoxography/tailwind-scrollbar ?